requirements.txt

```diff
@@ -1,3 +1,5 @@
 pyyaml>=5.3.0
 requests>=2.12.5
-tqdm>=4.59.0
\ No newline at end of file
+tqdm>=4.59.0
+gitpython>=3.1
```

Comment:
No need for `gitpython` now, use subprocess with `git` command instead in [git_utils.py](https://github.com/towhee-io/towhee/blob/main/towhee/utils/git_utils.py).

Reply:
fixed